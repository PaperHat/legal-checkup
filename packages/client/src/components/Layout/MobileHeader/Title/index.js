import styled from 'styled-components'

const Title = styled.div`
  width: 87px;
  height: 18px;
  font-family: MuseoSansCyrl;
  font-size: 20px;
  font-weight: 900;
  font-style: normal;
  font-stretch: normal;
  line-height: 1;
  letter-spacing: normal;
  color: #ffffff;
  margin-bottom: 20px;
  margin-top: -30px;
  margin-left: 5px;
`
Title.displayName = 'Title'

export default Title
