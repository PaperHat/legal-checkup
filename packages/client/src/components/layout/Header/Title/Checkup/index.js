import styled from 'styled-components'

import { MOBILE, WHITE, RED_ACCENT } from '../../../../../constants'

const Checkup = styled.div`
  color: ${({ format }) =>
    format === MOBILE ? `#${WHITE}` : `#${RED_ACCENT}`};
  line-height: 1;
  text-transform: uppercase;
  font-size: inherit;
`
Checkup.displayName = 'Checkup'

export default Checkup
