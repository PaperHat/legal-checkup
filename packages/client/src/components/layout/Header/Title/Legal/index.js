import styled from 'styled-components'

const Legal = styled.div`
  color: #fff;
  line-height: 1;
  text-transform: uppercase;
  font-size: inherit;
`
Legal.displayName = 'Legal'

export default Legal
