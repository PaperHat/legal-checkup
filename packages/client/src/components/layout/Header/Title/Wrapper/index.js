import styled from 'styled-components'
import styledMap from 'styled-map'

import { DESKTOP, TABLET, MOBILE } from '../../../../../constants'

const Wrapper = styled.div`
  font-family: 'Museo Sans Cyrl', sans-serif;
  font-weight: 900;
  line-height: 1;

  ${styledMap('format', { [`${DESKTOP}`]: `
      margin-left: 10px;
      font-size: 36px;
    `,
  [`${TABLET}`]: `
      margin-left: 10px;
      font-size: 25px;
    `,
  [`${MOBILE}`]: `
      margin-left: 8px;
      font-size: 18px;
    ` })};
`
Wrapper.displayName = 'Wrapper'

export default Wrapper
