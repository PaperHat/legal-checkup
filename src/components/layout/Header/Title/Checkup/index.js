import styled from 'styled-components'

import { MOBILE, WHITE, TITLE_TEXT } from '../../../../../constants'

const Checkup = styled.div`
  color: ${({ format }) => (format === MOBILE ? `${WHITE}` : `${TITLE_TEXT}`)};
  line-height: 1;
  text-transform: uppercase;
  font-size: inherit;
`
Checkup.displayName = 'StyledCheckup'

export default Checkup
