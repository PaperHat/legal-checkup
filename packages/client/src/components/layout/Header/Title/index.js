import * as React from 'react'

import Checkup from './Checkup'
import Legal from './Legal'
import Wrapper from './Wrapper'

export default function Title () {
  return (
    <Wrapper>
      <Legal>Legal</Legal>
      <Checkup>Checkup</Checkup>
    </Wrapper>
  )
}
