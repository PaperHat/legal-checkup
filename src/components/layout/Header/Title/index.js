import * as React from 'react'

import Checkup from './Checkup'
import Legal from './Legal'
import Wrapper from './Wrapper'

export default function Title ({ format }) {
  return (
    <Wrapper format={format}>
      <Legal>Legal</Legal>
      <Checkup format={format}>Checkup</Checkup>
    </Wrapper>
  )
}
