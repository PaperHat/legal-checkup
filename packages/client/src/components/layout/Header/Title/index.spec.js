import * as React from 'react'
import { create } from 'react-test-renderer'

import Title from './'

describe('components:layout:Header:Title', () => {
  it('matches the snapshot', () => {
    expect(create(<Title>Title</Title>).toJSON()).toMatchSnapshot()
  })
})
