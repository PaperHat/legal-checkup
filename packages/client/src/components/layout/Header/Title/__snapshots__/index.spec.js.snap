// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:layout:Header:Title renders correctly 1`] = `
.c2 {
  color: #b62645;
  line-height: 1;
  text-transform: uppercase;
  font-size: inherit;
}

.c1 {
  color: #fff;
  line-height: 1;
  text-transform: uppercase;
  font-size: inherit;
}

.c0 {
  font-family: 'Museo Sans Cyrl',sans-serif;
  font-weight: 900;
  line-height: 1;
  margin-left: 8px;
  font-size: 18px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    Legal
  </div>
  <div
    className="c2"
  >
    Checkup
  </div>
</div>
`;
